yaml/roads.yaml

```diff
@@ -1,5 +1,11 @@
 global:
+  - &output_properties
+    source: {col: source}
```

Comment:
Ah, right. Maybe add a comment up in the global section noting that? Something like

```
    # not all features (e.g. pistes) include names so names are added below
```